### PR TITLE
Update DMT protocol to support Run Hours & Odometer

### PR DESCRIPTION
**Pull request by jfunston83**
Added support for running time and odometer. 
They have a number of devices that track them, we use the G62's to track equipment hours.

### CODE REVIEW

**Comment by tananaev** [comment]
What is running time?

**Comment by jfunston83** [comment]
Hello, DMT calls it running time, it’s for tracking engine hours off of an ignition wire. There’s total hours, and trip hours. We use it for our customers that need to track running hours as well as location for generators and other equipment.

**Comment by tananaev** [comment]
We have attribute for engine hours.

**Comment by jfunston83** [comment]
Hi Anton, sorry, I missed that one. I updated it to use the engine hours attribute.

File: src/main/java/org/traccar/protocol/DmtProtocolDecoder.java [hunk 1]
```diff
@@ -188,6 +188,17 @@ private void sendResponse(Channel channel, int type, ByteBuf content) {
...
+                } else if (fieldId == 27) {
+
+                    position.set(Position.KEY_ODOMETER, buf.readUnsignedIntLE());
+                    position.set(Position.KEY_HOURS, buf.readUnsignedIntLE() * 1000);
+
+
+                } else if (fieldId == 26) {
+
+                    position.set(Position.KEY_ODOMETER_TRIP, buf.readUnsignedIntLE());
+                    position.set("TripHours", buf.readUnsignedIntLE() * 1000);
```
Comment (tananaev) [inline src/main/java/org/traccar/protocol/DmtProtocolDecoder.java]: Please use lower-camel-case standard that we use for all other attributes.

File: src/main/java/org/traccar/protocol/DmtProtocolDecoder.java [hunk 2]
```diff
@@ -188,6 +188,17 @@ private void sendResponse(Channel channel, int type, ByteBuf content) {
                     position.set(Position.KEY_OUTPUT, output);
                     position.set(Position.KEY_STATUS, status);
 
+                } else if (fieldId == 27) {
```
Comment (tananaev) [inline src/main/java/org/traccar/protocol/DmtProtocolDecoder.java]: Make sure you are order blocks in ascending order of the field ids.

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**